braviarc/braviarc.py

```diff
@@ -231,6 +231,10 @@ def set_volume_level(self, volume):
     def turn_on(self):
         """Turn the media player on."""
         self._wakeonlan()
+        # Try using the power on command incase the WOL doesn't work
+        POWER_ON_CMD = 'AAAAAQAAAAEAAAAuAw=='
```

Comment:
Anyway, doing that by asking the self.get_command_code method is better since it avoids to hardcode the command code in your code. Does it make sense?